pom.xml

```diff
@@ -527,7 +527,7 @@
                 </property>
             </activation>
             <properties>
-                <test.argLine>-Xmx1024m -Dfile.encoding=UTF-8</test.argLine>
+                <test.argLine>-Xmx1024m -Dfile.encoding=UTF-8 --add-opens java.base/java.lang=ALL-UNNAMED</test.argLine>
```

Comment:
@markpatton : Could you add a comment here to clarify why it's necessary to add `ALL-UNNAMED`?  I just would like to ensure we have documented why this is necessary in `dspace-7_x` (especially since it's not necessary in later releases).